decidim-participatory_processes/app/cells/decidim/participatory_processes/process_g_cell.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
+    # This cell renders the Grid (:g) process card
+    # for an given instance of a Process
+    class ProcessGCell < Decidim::CardGCell
+      private
+
+      def resource_path
+        Decidim::ParticipatoryProcesses::Engine.routes.url_helpers.participatory_process_path(model)
+      end
+
+      def resource_image_path
```

Comment:
This is also present in AssemblyGCell and ConferenceGCell

Reply:
Replied [there](https://github.com/decidim/decidim/pull/10219#discussion_r1086648699)